package.json

```diff
@@ -32,6 +32,7 @@
     "core-js": "^2.4.1",
     "crypto": "0.0.3",
     "debug": "~2.6.3",
+    "email-verify": "^0.2.1",
```

Comment:
Why is this added in a PR for CSS changes ?

Reply:
without installing this package the app won't build.I guess it is required to have the previous PR (#92) to be merged .